python/pyspark/sql/connect/functions.py

```diff
@@ -1098,6 +1098,13 @@ def array_intersect(col1: "ColumnOrName", col2: "ColumnOrName") -> Column:
 array_intersect.__doc__ = pysparkfuncs.array_intersect.__doc__
 
 
+def array_append(col1: "ColumnOrName", col2: "ColumnOrName") -> Column:
```

Comment:
I think we don't need to add it to Connect for now, we can do it in a separate PR with related tests in Connect.

Reply:
oh Sure ! Let me remove the changes being done for connect